enhancements/okd-on-fedora-coreos.md

```diff
@@ -32,128 +32,58 @@ superseded-by:
 - [ ] Graduation criteria for dev preview, tech preview, GA
 - [ ] User-facing documentation is created in [openshift/docs]
 
-## Open Questions [optional]
-
- 1. When is OCP migrating to Ignition config spec v3?
-
- Please see the Graduation Criteria section for more context on both open questions.
-
 ## Summary
 
 OKD is the OpenShift community distribution of Kubernetes.
 
-As part of the version 4 effort, the OKD working group has agreed to target Fedora CoreOS (FCoS) as the primary base operating system for OKD.
+As part of the version 4 effort, the OKD working group has decided to target Fedora CoreOS (FCOS) as the primary base operating system for OKD control plane and worker nodes.
 
-This document contains a proposal to create all OKD artifacts in OpenShift's Prow instance from the canonical OpenShift master and release branches alongside OCP artifacts.
-
-Eventually, container images for all components will be shared between OKD and OCP, with the exception of the machine-os-content container image which will be based on Fedora CoreOS content, and any container images that are based on RHEL images in OCP. These will be rebuilt on either a UBI, Fedora or CentOS base for OKD.
-
-A lot of this is already done. What's left is the un-forking of Machine Config Operator and (optionally) Installer repository and building variant-agnostic artifacts of them, as well as changing the Samples Operator to not go degraded on OKD clusters.
+This document contains a proposal to add missing FCOS support to key components (`machine-config-operator` and `installer`),
```

Comment:
We've also created OKD variants for `samples` (contains CentOS and UBI) and `marketplace` (community operators index can be pulled without a special pull secret). These are not critical and not sure if its worth mentioning those